src/mlpack/bindings/python/setup.py.in

```diff
@@ -41,11 +46,10 @@ else:
                   libraries=['mlpack', 'boost_serialization'],
                   library_dirs=['${CMAKE_BINARY_DIR}/lib/'],
                   # CMAKE_CXX_FLAGS seems to have an extra space.
-                  extra_compile_args=('-DBINDING_TYPE=BINDING_TYPE_PYX ' + \
-                      '-std=c++11${CMAKE_CXX_FLAGS}').split(' '),
-                  extra_link_args=['${OpenMP_CXX_FLAGS}'],
-                  undef_macros=[] if len("${DISABLE_CFLAGS}") == 0 \
-                      else '${DISABLE_CFLAGS}'.split(';')) \
+                  extra_compile_args=(extra_args,
```

Comment:
It looks like there is a syntax error here; if you can fix it, I think that it will work.